workinggroups/ci.md

```diff
@@ -0,0 +1,34 @@
+# CNCF CI WG Proposal
+
+## TOC Sponsor
+
+Camille Fournier
+
+## Objective
+
+Explore the intersection of cloud native and CI technology. Discuss options for taking some of the cluster resources and dedicating them to supporting an open source CI system that can be used by CNCF projects for their CI needs.
+
+## Goals and Expected Outcomes
+
+* We believe that it would be good for us to provide CI services to projects who need or want to use them
+* We need to understanding what, if any, SLA we can promise projects for this system
+* We need to scope what features this system will provide; there is some concern around trying to promise testing the full cross-product of integration with all of the different CNCF projects
+* We want to come away with a recommendation for staffing to support building out this initiative given project needs and desired SLA
```

Comment:
Are working groups expected to be long lived (e.g., still doing work in 20 years), working to a complete a task, or may there be cases of both? I ask because some of these, such as storage and networking I could see some day coming to a close while the CI one could go on indefinitely if the first bullet is true. It might be useful in the readme to describe expectations around and for the two types of WGs if they diverge on this point. If some are expected to close it might be useful to have a definition of done in the proposal.

Reply:
I agree. The CI WG is different kind of entity from the others. It is producing code for the CNCF that isn't a project unto itself, but is in service of the CNCF's projects.

Reply:
I agree that the CI WG proposal is a bit different from what all the WGs are currently doing.  If a group of people are needed to support CNCF, what about making a CNCF CI team -- or some other word that we can use to convey a very different kind of activity.